skbio/draw/tests/test_distributions.py

```diff
@@ -12,7 +12,10 @@
 
 import numpy as np
 import numpy.testing as npt
-import matplotlib.pyplot as plt
+try:
```

Comment:
Would it be worthwhile to create a utility context-manager with would handle the exception and skip it?